content-connector/src/main/java/dk/dbc/holdingitems/content/HoldingsItemsConnector.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright Dansk Bibliotekscenter a/s. Licensed under GNU GPL v3
+ *  See license text at https://opensource.dbc.dk/licenses/gpl-3.0
+ */
+
+package dk.dbc.holdingitems.content;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import dk.dbc.commons.metricshandler.CounterMetric;
+import dk.dbc.commons.metricshandler.MetricsHandlerBean;
+import dk.dbc.commons.metricshandler.SimpleTimerMetric;
+import dk.dbc.httpclient.FailSafeHttpClient;
+import dk.dbc.httpclient.HttpClient;
+import dk.dbc.httpclient.HttpGet;
+import net.jodah.failsafe.RetryPolicy;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.metrics.Metadata;
+import org.eclipse.microprofile.metrics.MetricType;
+import org.eclipse.microprofile.metrics.MetricUnits;
+import org.eclipse.microprofile.metrics.Tag;
+
+import javax.annotation.PostConstruct;
+import javax.ejb.Stateless;
+import javax.inject.Inject;
+import javax.ws.rs.InternalServerErrorException;
+import javax.ws.rs.ProcessingException;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.core.Response;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static javax.ws.rs.core.Response.Status;
+
+/**
+ * EJB to provide access to the HoldingsItems database.
+ */
+@Stateless
+public class HoldingsItemsConnector {
+    public static final Tag METHOD_TAG = new Tag("method", "getAgenciesThatHasHoldingsForId");
+    private static final Set<Integer> NO_RETRY_RESPONSES = Stream.of(Status.INTERNAL_SERVER_ERROR, Status.BAD_REQUEST).map(Status::getStatusCode).collect(Collectors.toSet());
```

Comment:
Set.of(Status.INTERNAL_SERVER_ERROR.getStatusCode(),
    Status.BAD_REQUEST.getStatusCode());
Kortere linje - lettere at læse?